src/Keywords/KeywordsInterface.php

```diff
@@ -33,77 +37,77 @@ public function setLanguage($language);
     /**
      * Returns Feature keywords (separated by "|").
      *
-     * @return string
```

Comment:
should be kept to indicate the native return type we'll add in the future.

Reply:
Just to be clear, is the point of this just to have the correct type for the future (and should therefore only add simple types)?

So for example:
```php
    /**
     * @phpstan-return TMultiLanguageKeywords
     */
    abstract protected static function getKeywordsArray(): array;
```
Should I add `@return array<string, whatever>` in that case?

Reply:
I would keep the simple case matching the future native type (which would be removed once we add a native return type). There is no point defining a precise type without type alias in `@return` while using an equivalent type alias in `@phpstan-return`. It only doubles the maintenance work (with phpstan checking only one of them)